src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -1514,6 +1565,8 @@ g.test('mip_levels')
       .combineWithParams(kMethodsToTest)
       .combine('format', kWorkingTextureFormats)
       .filter(formatCanBeTested)
+      .combine('dimension', ['2d', '3d'] as const)
```

Comment:
comment why we don't do 1d. Or better, comment about what dimensinos are tested in the description